server/server.go

```diff
@@ -1790,8 +1791,8 @@ func (s *Server) createClient(conn net.Conn) *client {
 
 	// Do final client initialization
 
-	// Set the First Ping timer.
-	s.setFirstPingTimer(c)
+	// Set the Ping timer. Will be reset once connect was received.
```

Comment:
Why not go back to `c.setPingTimer()` here instead so you don't need the new boolean in setFirstPingTimer().